Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/MicrosoftGraphMail_description.md

```diff
@@ -10,6 +10,16 @@ Depending on the authentication method that you use, the integration parameters
 In order to use Cortex XSOAR application and allow us access to O365 Outlook Mail (Using Graph API), an admin has to approve our app using an admin consent flow, by clicking on the following [link](https://oproxy.demisto.ninja/ms-graph-mail).
 After authorizing the Demisto app, you will get an ID, Token, and Key, which should be inserted in the integration instance configuration's corresponding fields.
 
+### Using National Cloud
+Using a national cloud endpoint is supported by setting the *Server URL* parameter to one of the following options:
+* US Government GCC-High Endpoint: `https://graph.microsoft.us`
+* US Government Department of Defence (DoD) Endpoint: `https://dod-graph.microsoft.us`
+* Microsoft 365 Germany Endpoint: `https://graph.microsoft.de`
+* Microsoft Operated by 21Vianet Endpoint: `https://microsoftgraph.chinacloudapi.cn`
+
+Please refer to [Microsoft Integrations - Using National Cloud](https://xsoar.pan.dev/docs/reference/articles/microsoft-integrations---authentication#using-national-cloud) for more information.
```

Comment:
i dont see have something national cloud in this article

Reply:
I have a PR for it. I didn't want to merge it before we merge this PR so it won't confuse customers.
https://github.com/demisto/content-docs/pull/821/files